packages/gestalt-codemods/generic-codemods/renameComponent.js

```diff
@@ -0,0 +1,82 @@
+// @flow strict
+
+/**
+ * CODEMOD to RENAME GESTALT COMPONENTS
+ * Ex. <Flyout /> to <Popover />
+ *
+ * TO RUN THIS CODEMOD
+ * yarn run:codemod renameComponent  <folder/file path> --previousCmpName=<value> --nextCmpName=<value>
```

Comment:
There's some inconsistency on shorthand vs fully-written out versions of "component". For clarity we should probably stick to the full version for these tools. However, we can probably sidestep that here: we're already in the context of the `renameComponent` codemod, so no need to repeat that. `previousName` and `nextName` should be fine.